lib/runcible/models/ostree_importer.rb

```diff
@@ -4,7 +4,7 @@ class OstreeImporter < Importer
       ID = 'ostree_web_importer'.freeze
       REPO_TYPE = 'OSTREE'.freeze
 
-      attr_accessor 'branches'
+      attr_accessor 'branches', 'depth'
```

Comment:
@jlsherrill good question. I guess on the katlelo side I am setting it to 0 by default so never noticed.